### ISSUE TITLE
Can add support kirikiri  tool of "AMBITIOUS MISSION" of extract ?

### ISSUE THREAD

**Issue by sum2012** [Kerisa/ExtractGal#1]
@Kerisa
Can add support kirikiri game of"AMBITIOUS MISSION" of extract ?
https://vndb.org/v33036
Thanks
(The game has sig protect)

process file [E:\test\\AMBITIOUS MISSION\ambitious_mission\data.xp3]:
[failed] create error [倀].
[failed] create error [startup.tjs].
[failed] create error [倁].
[failed] create error [倂].
[failed] create error [倃].
[failed] create error [倄].
[failed] create error [倅].
[failed] create error [倆].
[failed] create error [倇].
[failed] create error [倈].
[failed] create error [倉].
[failed] create error [倊].
[failed] create error [個].
[failed] create error [倌].
[failed] create error [倍].
[failed] create error [倎].
[failed] create error [倏].
[failed] create error [倐].
[failed] create error [們].
[failed] create error [倒].
[failed] create error [倓].
[failed] create error [倔].
[failed] create error [倕].
[failed] create error [倖].
[failed] create error [倗].
[failed] create error [倘].

**Comment by Kerisa** [comment]
Thanks for the information, will take a look.

**Comment by Kerisa** [comment]
I have to say it's been quite challenging to directly extract resources like before. After a few days, the only approach I could come up with is passively extracting resources during gameplay. On the bright side, this method doesn't require adapting to different encryptions.

**Comment by sum2012** [comment]
![1](https://github.com/Kerisa/ExtractGames/assets/2177532/294f9f73-9fc4-444c-9848-a8c57cb9becc)
The game crash even use --passive

**Comment by Kerisa** [comment]
Did you lanunch it with `Loader_d.exe ambitious_mission.exe --passive`? `--passive` should be followed right after `.exe`, and you won't get that `Begin extracting, wait a minute....` message popup

**Comment by sum2012** [comment]
Yes, but this time do not pop up "Begin extracting, wait a minute...." and produce wrong filename
![1](https://github.com/Kerisa/ExtractGames/assets/2177532/6dee0933-e72e-4ad5-9053-e3f2c186f5fe)

**Comment by Kerisa** [comment]
I did not filter these files, you can just ignore them. And if sort the files by time, you can see that each garbled file essentially lines up with a decrypted file.
![图片](https://github.com/Kerisa/ExtractGames/assets/10655671/924b3894-532d-44b1-988a-1441c306050a)
